hummingbird/ml/operator_converters/skl_sv.py

```diff
@@ -0,0 +1,117 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import torch
+import scipy
+import numpy as np
+
+from onnxconverter_common.registration import register_converter
+
+"""
+Converters for scikit-learn linear models: SVC, NuSVC.  (LinearSVC is covered by linear_classifier.py)
+"""
+
+
+class SVC(torch.nn.Module):
+    def __init__(self, kernel, degree, sv, nv, a, b, gamma, coef0, classes, device):
+        super(SVC, self).__init__()
+        self.kernel = kernel
+        self.degree = degree
+        self.gamma = gamma
+        self.regression = False
+        sv = sv.toarray() if type(sv) == scipy.sparse.csr.csr_matrix else sv
+
```

Comment:
Any specific reason why we have space here? Otherwise we can just remove it.

Reply:
the sv does seem a bit lost.